docs/user/site.rst

```diff
@@ -496,9 +497,20 @@ The following flags will add multiple packages:
   - *gluon-web-network*
   - *gluon-web-wifi-config*
   - *gluon-web-autoupdater* (if the *autoupdater* feature is enabled)
-  - *gluon-web-mesh-vpn-fastd* (if the *mesh-vpn-fastd* feature is enabled)
+.. TODO: this seems to be missing somehow: 
```

Comment:
Soething was wrong here.

can this line be deleted? or is something missing in `packages/features`?

Reply:
Because in that line 11 there is

    gluon-config-mode-mesh-vpn

Not

    gluon-web-mesh-vpn-fastd

Reply:
Because in that line 11 there is

    gluon-config-mode-mesh-vpn

Not

    gluon-web-mesh-vpn-fastd